### ISSUE TITLE
Changes to make for Washington survey

### ISSUE THREAD

**Issue by darcyrao** [statnet/WHAMP#63]
- Ask about alter race as a multiple response variable? _Advantage_: can define alter race in the same way as ego race, including grouping men who are black alone or in combination with another race together. _Disadvantage_: Men may not know all the racial/ethnic categories their partner identifies as. But I think this is almost equally an issue with a single option response. We could add an option along the lines of "Mixed, but I'm not sure which groups"

- Ask about total number of one-time partnerships in the past 12 months

- Ask about cohabitation?

**Comment by martinamorris** [comment]
> Ask about alter race as a multiple response variable? Advantage: can define alter race in the same way as ego race, including grouping men who are black alone or in combination with another race together. Disadvantage: Men may not know all the racial/ethnic categories their partner identifies as. But I think this is almost equally an issue with a single option response. We could add an option along the lines of "Mixed, but I'm not sure which groups"

Good idea, but the latter option would leave us with some difficult choices when we try to characterize mixing -- since the ego and alter options are different.

> Ask about total number of one-time partnerships in the past 12 months (this measure was omitted from the 2017-2018 national ART-NET survey)

Done.

> Ask about cohabitation?

Done, also asking about marriage now

> Consider making changes to measures of condom use (see issue #30).

This is important, but also, as you note, a fairly complicated knot of issues. 

In addition to the context rabbit hole, I have some concerns about asking the question in the abstract (e.g., "I always use condoms with main partners"), rather than in the concrete context of a specific partner.  Even in this concrete context, we end up asking the question about frequency in the abstract ("how often did you use condoms with this partner").  The alternative preferred by many in the demography field is to ask about the last event.  This prioritizes specificity and clarity, at the cost of measuring heterogeneity (within persons and partnerships).  Heterogeneity is instead measured at the population level, as rates of use broken down by different types of partnerships (and possibly nodal attributes), and these probabilities apply to everyone in the category.

Worth a discussion, and I'll ask @smjenness to weigh in here.

**Comment by smjenness** [comment]
1. Race: we are collapsing everything down into Black/White/Hisp/Other in practice, with multiple races going into Other. It's a very small group for the indexes and would be more so for the partners. 

2. Marriage/cohabitation: ok

3. Condoms: I realize there's many different ways to ask about this. Unless there is a compelling reason to deviate from the current ART-Net approach based on the research questions of WHAMP, we should not change to maintain comparability with our existing data.

**Comment by darcyrao** [comment]
@smjenness, for my model, we categorized men who reported black race in combination with any other race as black and defined three racial/ethnic groups (Hispanic, black, and other). This was in part motivated by the small number of black respondents in Washington, and by some articles I found that suggest that the experiences and identity of men who are mixed race with black as one of the racial groups are probably more similar to black men than to white men (who make up the vast majority of the "other" group). I think it's better to give respondents the option to list multiple racial groups for their partners - if they think their partner is mixed race but aren't sure which groups, they will probably either put "other" or guess based on appearance. If they choose "other", we'd be in the same boat as before since this would be grouped the same way as if they had been given a "multiple" response option. I also think it's better to measure alter race in the same way we measure ego race.
